checkov/kubernetes/image_referencer/provider/k8s.py

```diff
@@ -90,6 +54,20 @@ def extract_images_from_containers(containers: Any) -> list[str]:
     return image_names
 
 
+def _extract_images_from_spec(spec: dict[str, Any]) -> list[str]:
+    image_names: list[str] = []
```

Comment:
how about making this a `set()` then you don't need to transform it to a set and back to list 🙂

Reply:
good idea - fixed

Reply:
maybe all of the functions here can return sets ?

Reply:
also thought the same, but I think it is a bigger change, because of the typing mismatch.

Reply:
Just didn't watch to change api